docs/setup-guide/build_k8s_from_source/setup_1.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
```

Comment:
add

```bash
set -e
```

to [stop the script as soon as a failure happens](https://renenyffenegger.ch/notes/Linux/shell/bash/built-in/set/e#:~:text=set%20%2De%20causes%20a%20script,equivalent%20to%20set%20%2Do%20errexit%20.)

Reply:
`set -e` was added in all three scripts.